drivers/gpio/gpio_sx1509b.c

```diff
@@ -217,6 +218,13 @@ static int gpio_sx1509b_config(struct device *dev, int access_op, u32_t pin,
 				    SX1509B_REG_OPEN_DRAIN,
 				    pins->open_drain);
 
+	if (ret)
```

Comment:
use brackets around if statement

Reply:
I fixed it but kind of made the pull request messy. Maybe there's a way to squash it before merge or should I create a new pull request?

Reply:
There is no need to create a new PR. We work by rewriting local history on a branch from which PR was created and force pushing the branch to Github. Some more info here [Contribution Workflow](https://docs.zephyrproject.org/latest/contribute/index.html#contribution-workflow).

The coding style requires us to use braces even for single-line code blocks. Since `gpio_sx1509b.c` already violates this rule in many places the proper way to issue this PR would be to submit it as two commits. The first one should fix all the existing violations to the 'braces' policy and the second should add support for polarity with the correct usage of braces.

Reply:
Thanks. I'm still trying to wrap my head around the workflow.

I understand. Conveniently others fixed the braces after my PR. I merged the changes with my branch before fixing the PR. Although, am I right to assume that updating `gpio_sx1509b.c` in my branch wasn't really needed in order to merge PR properly?

Reply:
Since others have already fixed remaining brackets, indeed you'll only need to push one commit containing your original changes. But in this specific case they still need to be applied on the latest version of the file or otherwise there would be merge conflicts. The other and your commit are modifying overlapping lines of code.

There are different ways to handle your problem, I would probably do it this way:
```
git checkout sx1509b
git reset master
```
assuming sx1509b tracks master branch. At this point you should only see your modifications and `git status` should say:
```
On branch sx1509b
Your branch is up to date with 'master'.
```
All your history for the last 3 commits is lost. You will need to re-type your original commit message. Now:
```
git add drivers/gpio/gpio_sx1509b.c
git commit -s
git push -f origin sx1509b
```

In the future, if you are faced with similar problem don't use `git merge` but rather `git rebase`. By default `git rebase` will put whatever work you have done on the latest version of the branch you track. In the process you may need to resolve merge conflicts.

Reply:
Thank you very much for your detailed explanation!
I hope this PR is now suitable for merging.

Reply:
Looks good, thanks! Now we need to wait for @nashif to merge it.